core/functions.core.php

```diff
@@ -277,7 +277,11 @@ function redirect($dest)
  */
 function array_extract($ary, $columnKey)
 {
-    return array_map(create_function('&$row', 'return $row["'.$columnKey.'"];'), $ary);
```

Comment:
did you check if array_column wouldn't do the job?

Reply:
this can be dropped now?

Reply:
yes its removed now